bbot/scanner/scanner.py

```diff
@@ -55,10 +57,12 @@ def __init__(
         if modules is None:
             modules = []
         if output_modules is None:
-            output_modules = ["human"]
+            output_modules = ["python"]
```

Comment:
Are events still printed to stdout when bbot defaults to using the python output module?

Reply:
No, they are yielded but not printed. Also this code should answer your first question. The python output module is default.